### DISCUSSION TITLE
Tiptap does not build because of an 'unexpected token' error when trying to incorporate it into Laravel Nova

### DISCUSSION THREAD

**Discussion by gavinhewitt** [ueberdosis/tiptap#815] [General Questions] [answered]
Hi,

I'm trying to create a custom tiptap for Laravel Nova but I keep on getting these errors no matter what solution I try. The culprit is the spread operator (...) and I've tried to fix it by adding a .babelrc with these settings, but no luck. Not sure what I could try next. I looked at @manogi/nova-tiptap and used the same settings/package.json to see if that would solve it, same errors :-(

Help appreciated!

```
{
    "presets": ["@babel/preset-env"],
    "plugins": ["@babel/plugin-proposal-object-rest-spread", "transform-object-rest-spread"]
}  
```

![image](https://user-images.githubusercontent.com/1969103/91641004-df560b80-ea21-11ea-8575-6c62283a245c.png)

**Comment by gavinhewitt** [comment, marked as answer]
Got the builds working by upgrading Laravel Mix to the latest version, which makes use of the @babel/plugin-proposal-object-rest-spread plugin. Awesome!

However, the field is not showing. No errors or warnings whatsoever. Does anyone know why this could be happening?

**Reply by hanspagel** [reply]
Thanks for sharing! Did you try the existing Laravel Nova field by @manogi?

https://github.com/manogi/nova-tiptap

Maybe that helps!

**Reply by gavinhewitt** [reply]
Hi Hans,

Yes, I did. But as I need more customisation that package is not gonna work for me. I did however use it as a blueprint and I started by using the same settings in the package.json, which gave me this error. The error is fixed but the field is not showing.

**Reply by gavinhewitt** [reply]
I also reached out to the creator to ask if he has any clue as to why the field won't show.

**Reply by gavinhewitt** [reply]
Fixed!

This made me realize I could just rebuild the nova files to be able to check Vue devtools (and see if there were possible errors etc): 
JeffreyWay/laravel-mix#198

**Reply by hanspagel** [reply]
Thanks for sharing!
